commit 9f0791091868f449c8933902b556652e6c24875c (refs/changes/73/4873/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-09-25T18:39:26+00:00 (1 year ago)
    
    TRITON-786 dapi needs to learn about "HVM API" from sysinfo
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/cnapi/server.js b/lib/cnapi/server.js
index 14addaa..18e9920 100644
--- a/lib/cnapi/server.js
+++ b/lib/cnapi/server.js
@@ -206,6 +206,7 @@ var s_sysinfo = {
 		'CPU Physical Cores': base.intg.pos.req,
 		'UUID': base.str.uuid.req,
 		'Hostname': base.str.hostname.opt,
+		'HVM API': base.bool.opt,
 		'CPU Total Cores': base.intg.pos.req,
 		'MiB of Memory': base.intg.pos.req,
 		'Zpool': base.str.nonempty.req,
diff --git a/lib/dapi/server.js b/lib/dapi/server.js
index 9d376fb..27cc5f5 100644
--- a/lib/dapi/server.js
+++ b/lib/dapi/server.js
@@ -50,6 +50,7 @@ var s_sysinfo = {
 		'Bhyve Capable': base.bool.opt,
 		'Bhyve Max Vcpus': base.intg.pos.opt,
 		'CPU Total Cores': base.intg.pos.req,
+		'HVM API': base.bool.opt,
 		'Network Interfaces': s_si_nics_r,
 		'Live Image': base.str.nonempty.req,
 		'SDC Version': base.str.nonempty.opt
diff --git a/package.json b/package.json
index c3bb13e..0046cc2 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "joyent-schemas",
 	"description": "Joyent JSON Schemas",
-	"version": "0.3.1",
+	"version": "0.3.2",
 	"author": "Joyent (joyent.com)",
 	"main": "lib/index.js",
 	"repository": {
